src/components/NewWorkspaceModal.js

```
@@ -83,6 +83,12 @@ const NewWorkspaceModal = withDisplayName('NewWorkspaceModal', ({
     setIsAlphaRegionalityUser(await Ajax(signal).Groups.group(getConfig().alphaRegionalityGroup).isMember())
   })
 
+  const createLeoApp = withErrorIgnoring(async workspace => {
```

It might be good to add a comment as to why you are ignoring the error.

---

added!